modules/seqexec/web/client/src/main/scala/seqexec/web/client/components/tabs/CalibrationQueueTab.scala

```diff
@@ -98,10 +95,10 @@ object CalibrationQueueTab {
       ^.onDragOver ==> { (e: ReactDragEvent) =>
         e.preventDefaultCB *> Callback { e.dataTransfer.dropEffect = "copy" }
       },
-      ^.onDragEnter ==> b.runStateFn(onDragEnter),
-      ^.onDrop ==> b.runStateFn(onDrop),
-      ^.onDragEnd --> b.runState(onDragEnd),
-      ^.onDragLeave --> b.runState(onDragEnd),
+      ^.onDragEnter ==> onDragEnter(b) _,
+      ^.onDrop ==> onDrop(b) _,
+      ^.onDragEnd --> onDragEnd(b),
+      ^.onDragLeave --> onDragEnd(b),
```

Comment:
What is the difference between `==>` and `-->`?

Reply:
In one case `-->` you only pass a `Callback` which essentially is the effect you want to execute.
`==>` is similar but you also get the `ReactEvent` which maybe useful is some cases
It is documented here
https://github.com/japgolly/scalajs-react/blob/master/doc/VDOM.md#event-handlers

Reply:
Thanks!